superset/advanced_data_type/plugins/internet_port.py

```diff
@@ -48,6 +48,11 @@
     "ldaps": [636],
     "imap2": [143],  # aka imap
     "imaps": [993],
+    "MySQL": [3306],
+    "Remote Desktop": [3389],
+    "dns": [53],
+    "ctf": [84],
+    "pdap": [344] 
```

Comment:
This is still failing:
```suggestion
    "pdap": [344],
```